.github/workflows/test.yml

```diff
@@ -3,7 +3,30 @@ on:
   pull_request:
 
 jobs:
+  filter_paths:
+    runs-on: ubuntu-20.04
+    outputs:
+      lint_files: ${{ steps.filter.outputs.lint_files }}
+      fixer_all: ${{ steps.filter.outputs.fixer_all }}
+      rector_all: ${{ steps.filter.outputs.rector_all }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Filter paths
+        id: filter
+        uses: dorny/paths-filter@v3
+        with:
+          filters: |
+            lint:
+              - added|modified: '**.php'
+            fixer_all:
+              - 'composer.json'
+            rector_all:
+              - 'composer.json'
+
```

Comment:
`list-files: 'shell'` is required.
https://github.com/dorny/paths-filter?tab=readme-ov-file#usage

Reply:
Thank you for your comment.
I have corrected it in commit [a31eca0](https://github.com/litalico-engineering/eg-r2/pull/23/commits/a31eca0a57c7bf542e167088ee229eeff45a86d3)